mne/decoding/receptive_field.py

```diff
@@ -0,0 +1,353 @@
+import numpy as np
+from .base import _get_final_est, BaseEstimator, _check_estimator
+
+
+class ReceptiveField(BaseEstimator):
```

Comment:
cool, added in latest push + modified examples to use `rf.times_`. Think that's OK from an API standpoint? e.g., on `fit` it will create `delays_` which is delays in samples, and `times_` which is delays in seconds.